x/ethbridge/types/msgs.go

```diff
@@ -79,16 +79,28 @@ func (msg MsgLock) GetSigners() []sdk.AccAddress {
 }
 
 // GetProphecyID get prophecy ID for lock message
-func (msg MsgLock) GetProphecyID(doublePeggy bool, sequence, globalNonce uint64, tokenAddress string) []byte {
+func (msg MsgLock) GetProphecyID(
```

Comment:
Not required for this PR.

Have we decided against merging these two functions?

Reply:
you means GetProphecyID for MsgLock and MsgBurn?

Reply:
without the generics support, let's just leave it as is.